packages/FormElement/stories/examples/NestedExample.js

```diff
@@ -9,63 +12,80 @@ export default function NestedExample() {
   const isReadOnly = false;
 
   return (
-    <FormElement hasFieldSet label="Form Legend">
-      <FormElement.Content>
-        {({ ariaDescribedBy: ariaDescribedByOuter }) => (
-          <>
-            <FormElement style={{ marginTop: stylers.spacer(2) }} isInline label="Sub Label">
-              <FormElement.Content>
-                {({ idForLabel, ariaDescribedBy }) => (
+    <Fieldset>
+      <LayoutFlexParent>
+        <LayoutLeftCol>
+          <Fieldset.Label>Form Legend</Fieldset.Label>
+        </LayoutLeftCol>
+        <LayoutRightCol>
+          <Fieldset.Instructions>Some instructions for field set Element</Fieldset.Instructions>
+          <Fieldset.Content>
+            <FormElement style={{ marginTop: stylers.spacer(2) }}>
+              <Label
+                help={
                   <>
-                    <input
-                      aria-required={hasRequiredLabel}
-                      aria-describedby={ariaDescribedBy + ariaDescribedByOuter}
-                      disabled={isDisabled}
-                      id={idForLabel}
-                      readOnly={isReadOnly}
-                    />
-                    <input
-                      aria-required={hasRequiredLabel}
-                      aria-describedby={ariaDescribedBy + ariaDescribedByOuter}
-                      disabled={isDisabled}
-                      readOnly={isReadOnly}
-                    />
+                    Give me some help. <a href="wegalvanize.com">Learn more</a>.
                   </>
-                )}
-              </FormElement.Content>
-              <FormElement.Error>{errorText}</FormElement.Error>
-              <FormElement.Help>
-                Give me some help. <a href="wegalvanize.com">Learn more</a>.
-              </FormElement.Help>
+                }
+              >
+                Sub Label
+              </Label>
+              <Content>
+                {a11yProps => {
+                  return (
+                    <>
+                      <input
+                        aria-required={hasRequiredLabel}
+                        disabled={isDisabled}
+                        readOnly={isReadOnly}
+                        id={a11yProps.id}
+                        aria-describedby={a11yProps["aria-describedby"]}
+                      />
+                      <input
+                        aria-required={hasRequiredLabel}
+                        disabled={isDisabled}
+                        readOnly={isReadOnly}
+                        aria-describedby={a11yProps["aria-describedby"]}
+                      />
```

Comment:
Is there something special you're trying to illustrate with these native `<inputs>` as opposed to the Paprika `<Inputs>` below?

Reply:
Not really, just showing the difference when using each type, native or Paprika Input